frontend/src/scenes/project-homepage/recentInsightsLogic.ts

```diff
@@ -0,0 +1,32 @@
+import { kea } from 'kea'
+
+import { teamLogic } from 'scenes/teamLogic'
+import { InsightModel } from '~/types'
+import api from 'lib/api'
+
+import type { recentInsightsLogicType } from './recentInsightsLogicType'
+export const recentInsightsLogic = kea<recentInsightsLogicType>({
+    path: ['scenes', 'project-homepage', 'recentInsightsLogic'],
+    connect: {
+        values: [teamLogic, ['currentTeamId']],
+    },
+    loaders: ({ values }) => ({
+        recentInsights: [
+            [] as InsightModel[],
+            {
+                loadRecentInsights: async () => {
+                    const response = await api.get(
+                        `api/projects/${values.currentTeamId}/insights/?my_last_viewed=true&order=-my_last_viewed_at`
```

Comment:
I think we're moving away from using the currentTeamId pattern and towards the api builder which handles that for us. Could we create another entry in `frontend/src/lib/api.ts` for `insights`?

Reply:
Didn't know this new pattern was a thing! I like it.

I took a look at moving `/insights` into it, and it looks like a pretty good refactor as we use the endpoint all over. Thoughts on it being a separate PR?

Reply:
That sounds totally good to me! 👍